tutorwordpress/plugin.py

```diff
@@ -84,13 +200,28 @@ def config(context: click.Context) -> None:
     Print out the configurations used in WordPress plugin.
```

Comment:
I suggest converting this to a `do wordpress config` command. That way you have access to the template system and you don't have to manually load the project configuration.

Reply:
I thought that do commands used as jobs, running inside containers. I can write it like do Wordpress config but if I don't return an iterable of service and command, it will fail.

Reply:
yeah, I guess you're right.